src/Language/Python/Render.hs

```diff
@@ -0,0 +1,10 @@
+module Language.Python.Render
+  ( -- * Common Functions
+    showModule, showStatement, showExpr
+    -- * Rendering
+  , RenderOutput, showRenderOutput, singleton, cons
```

Comment:
I see `RenderOutput` & co. as an implementation detail. Is it helpful to have it exposed here?